lib/moxl/src/Moxl/Xec/Action/MAM/Get.php

```diff
@@ -40,6 +40,18 @@ public function setBefore($before = true)
         return $this;
     }
 
+    public function setStart($start)
```

Comment:
I don't really get the need of those two methods, this piece of code is already taking care of that https://github.com/movim/movim/blob/master/lib/moxl/src/Moxl/Xec/Action.php#L50 :p

Reply:
Ooooh. That explains a lot xD